bluesky/run_engine.py

```diff
@@ -1560,6 +1560,17 @@ def _save(self, msg):
 
         # Event Descriptor documents
         desc_key = self._bundle_name
+
+        # This is a separate check because it can be reset on resume.
+        seq_num_key = desc_key
+        if seq_num_key not in self._sequence_counters:
+            counter = count(1)
+            counter_copy1, counter_copy2 = tee(counter)
+            self._sequence_counters[seq_num_key] = counter_copy1
+            self._teed_sequence_counters[seq_num_key] = counter_copy2
+        self._bundling = False
+        self._bundle_name = None
```

Comment:
line 1601 needs `self._bundle_name`
I recommend:
`bundle_name = self._bundle_name`
...

Reply:
indeed, we already had grabbed this a few lines above as `desc_key`.

Reply:
yes! looks good to me